server/src/main/kotlin/suwayomi/tachidesk/server/ServerSetup.kt

```diff
@@ -161,25 +171,40 @@ fun applicationSetup() {
     LocalSource.register()
 
     // create system tray
-    if (serverConfig.systemTrayEnabled) {
+    serverConfig.subscribeTo(serverConfig.systemTrayEnabled, { systemTrayEnabled ->
         try {
-            systemTrayInstance
+            if (systemTrayEnabled) {
+                systemTrayInstance
+            } else {
+                systemTrayInstance?.remove()
+            }
```

Comment:
I believe this will break if we disable system tray and then try to enable it again.

Reply:
yeah... testing it correctly would be helpful
it can be removed but trying to show it again does nothing, but that just because the setup has to be done again and just writing the instance here does nothing ofc..., it has to be recreated then it works

Reply:
fixed